lib/hashtable.cc

```diff
@@ -122,6 +122,63 @@ consume_fasta(
     delete parser;
 }
 
+void Hashtable::consume_fasta_bitsplit(
```

Comment:
I agree the name is wonky. `consume_fasta_sampling` is better but honestly I've thought of half a dozen names and I'm not in love with any of them.